components/shooter.py

```diff
@@ -134,11 +140,7 @@ def execute(self) -> None:
         """This gets called at the end of the control loop"""
         inclinator_speed = self.inclinator_controller.calculate(
             self._inclination_angle(),
-            clamp(
-                self.desired_inclinator_angle,
-                ShooterComponent.MIN_INCLINE_ANGLE,
-                ShooterComponent.MAX_INCLINE_ANGLE,
-            ),
```

Comment:
Since inclinator angle is a tunable it can be set to values outside of the lookup table during testing, I think we want this otherwise redundant check to prevent mechanism damage in this case.